cache.go

```diff
@@ -0,0 +1,13 @@
+package links
+
+type CacheStore interface {
```

Comment:
We don't need an interface unless there are two or more implementations of it. How about just `type Cache map[string]bool`?